src/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>iAmWaris97 | Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <!-- parent container -->
+  <div class="container">
+
+    <!-- navbar -->
+    <nav class="navbar">
+      <a class="logo" href="#home">
+        <img src="./assets/img/my-logo.svg" alt="iAmWaris97-logo">
+      </a>
+
+      <a class="nav-toggler" href="#menu">
+        <img src="./assets/img/nav-toggler.svg" alt="">
+      </a>
+    </nav>
+
+    <!-- headline container -->
+    <section class="headline">
+
+      <h2 id="head-primary">
+        <span>iAmWaris97</span><span>Glad to see</span><span>you!</span>
+      </h2>
+
+      <p id="head-intro">I’m a software developer! I can help you build a product , feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.
+      </p>
+
+      <div class="social-media">
+        <p class="lets-connect">
+          LET’S CONNECT
+        </p>
```

Comment:
- [ ] It is important to note that this is supposed to be a link that will direct the user to the section that you will be building later in the next milestones. That is why I would suggest you use the anchor tag `<a>`. But keep the styling. It looks great 🔥.